tests/factories.py

```diff
@@ -26,6 +26,7 @@ class Meta:
     build_id = factory.sequence(lambda n: n + 100)
     name = factory.Faker("word")
     tag = factory.RelatedFactory(SoftwareBuildTagFactory, factory_related_name="software_build")
+    completion_time = datetime.now()
```

Comment:
This looks like it's generating a bunch of warnings in the test output. I would use Django's timezone.now() function to automatically generate TZ-aware datetimes using whatever timezone is configured in settings.